api/v2/openapi.yaml

```diff
@@ -451,6 +456,38 @@ definitions:
           - annotations
           - status
       - $ref: '#/definitions/alert'
+  extendedGettableAlert:
+    allOf:
+      - type: object
+        properties:
```

Comment:
I've had to duplicate all of these properties due to a bug in swagger generation for Elm that occurs when using inheritance.